.github/workflows/serenity-coverage.yml

```diff
@@ -0,0 +1,159 @@
+name: Nightly SerenityOS Coverage
+
+on:
+  # Automatically run at the end of every day.
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
Given the sad duplication here, it might make sense to change the lagom/serenity workflows into 'reusable workflows' with the on: workflow_call  trigger

https://docs.github.com/en/actions/using-workflows/reusing-workflows#creating-a-reusable-workflow

Reply:
aw man my google-foo is terrible, I looked for GH's equivalent for azure templates and did not find this at all.

Testing this over at https://github.com/trflynn89/serenity/tree/nightly_pipelines2 before pushing to this branch